client/deis.py

```diff
@@ -824,6 +833,7 @@ def auth_logout(self, args):
         self._settings['controller'] = None
         self._settings['username'] = None
         self._settings['token'] = None
+        self._settings['ssl_verify'] = None
```

Comment:
this is starting to get a bit unwieldly. Would you mind changing it to the following?

```python
for i in ['controller', 'username', 'token', 'ssl_verify']:
    self._settings[i] = None
```